pages/main/install.md

```diff
@@ -103,6 +98,28 @@ If you have questions regarding the installation process, please check out the *
   <i>Support: <span class="star-good"><i class="icon ion-md-star"></i><i class="icon ion-md-star"></i></span><span class="star-bad"><i class="icon ion-md-star"></i><i class="icon ion-md-star"></i><i class="icon ion-md-star"></i></span></i>
 </div></a>
 
+<div class="page-header">
+  <h1>Experimental</h1>
+</div>
+
+The AsteroidOS support level for some watches is currently deemed experimental for various reasons. These watches are not recommended for daily use. Despite yet to be fixed major problems, they might still be able to give an overall impression of AsteroidOS.
```

Comment:
"Despite yet to be fixed major problems"

Sounds like an odd sentence ?

Reply:
May as well drop it?

```
They are not recommended for daily use but should still give an overall impression of AsteroidOS.
```

Reply:
Agreed

Reply:
sgtm